frontend/src/core/models/task/task-create-view-model.ts

```diff
@@ -0,0 +1,7 @@
+export interface TaskCreateViewModel {
+    currentUser?: number;
+    project?: [string, string, number];
```

Comment:
Why array insted of interface ?

Reply:
It's not my code. I just imported it to my file to resolve conflicts. When anyone works on it, he will figure out how to use it.